ferrocene/tools/self-test/src/error.rs

```diff
@@ -17,12 +17,13 @@ pub(crate) enum LinkerArgsErrorKind {
 }
 
 #[derive(Debug)]
+#[allow(clippy::enum_variant_names)]
 pub(crate) enum Error {
     NoSysroot,
     MissingBinary { directory: PathBuf, name: String },
     WrongBinaryPermissions { path: PathBuf },
     MetadataFetchFailed { path: PathBuf, error: std::io::Error },
-    VersionFetchFailed { binary: String, error: CommandError },
+    VersionFetchFailed { binary: String, error: Box<CommandError> },
```

Comment:
why is clippy suggesting this

Reply:
It's the `result_large_err` lint: https://rust-lang.github.io/rust-clippy/master/index.html#/result_large_err